config/dca/cadesGPUCondo.cmake

```diff
@@ -14,9 +13,9 @@ set(PYCICLE_MACHINE "local")
 set(PYCICLE_ROOT "/lustre/or-hydra/cades-cnms/epd/DCA_GPU_CI")
 # a flag that says if the machine can send http results to cdash
 set(PYCICLE_HTTP TRUE)
-# Method used to launch jobs "slurm", "pbs" or "direct" supported
-set(PYCICLE_JOB_LAUNCH "pbs")
-
+# Launch jobs using slurm rather than directly running them on the machine
+set(PYCICLE_SLURM FALSE)
+set(PYCICLE_PBS TRUE)
```

Comment:
I think the `set(PYCICLE_JOB_LAUNCH "pbs")` approach is nicer than having a bool, for each possiblity. This was changed in my last big update and I suspect you started adding these tweaks before I merged your other stuff.

Reply:
I didn't test the dca master branch CI so I missed this.